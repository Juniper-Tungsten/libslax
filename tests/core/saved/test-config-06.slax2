/* Machine Crafted with Care (tm) by slaxWriter */
version 1.1;

output-method xml {
    indent "yes";
    media-type "image/svg";
}

match configuration {
    <top> {
        for-each (interfaces/interface) {
            sort name;
            copy-of .;
        }
    }
}
