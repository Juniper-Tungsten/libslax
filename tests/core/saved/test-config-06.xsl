<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
  <xsl:output method="xml" indent="yes" media-type="image/svg"/>
  <xsl:template match="configuration">
    <top>
      <xsl:for-each select="interfaces/interface">
        <xsl:sort select="name"/>
        <xsl:copy-of select="."/>
      </xsl:for-each>
    </top>
  </xsl:template>
</xsl:stylesheet>
