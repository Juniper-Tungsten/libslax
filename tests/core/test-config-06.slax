version 1.0;

<xsl:output method="xml" indent="yes" media-type="image/svg">;

match configuration {
    <top> {
	for-each (interfaces/interface) {
	    <xsl:sort select="name">;
	    copy-of .;
	}
    }
}
