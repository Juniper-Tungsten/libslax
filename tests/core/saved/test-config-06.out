<?xml version="1.0"?>
<top>
  <interface>
      <name>ge-1/1/1</name>
    </interface>
  <interface>
      <name>xe-1/1/1</name>
    </interface>
</top>
